homeassistant/data_entry_flow.py

```diff
@@ -470,6 +471,7 @@ class FlowHandler:
 
     # Set by developer
     VERSION = 1
+    MINOR_VERSION = 1
```

Comment:
It's because:
- `VERSION` defaults to 1, not to 0
- [`minor_version` of the store helper defaults to 1, not to 0](https://github.com/home-assistant/core/blob/216c3d73d8264d2c85369bc76f9cbedd3696bf81/homeassistant/helpers/storage.py#L95)

Reply:
Aah, I did not know the store helper was abstract